application/controllers/UserRoleController.php

```diff
@@ -38,56 +48,49 @@ public function index()
             $model->setAttributes($aPermissiontemplatesParam, false);
         }
 
-        $this->_renderWrappedTemplate(
-            null,
-            'permissiontemplates/index',
-            array(
+        // Green Bar (SurveyManagerBar) Page Title
+        $aData['pageTitle'] = gT('User roles');
+
+        //this is really important, so we have the aData also before rendering the content
+        $this->aData = $aData;
+
+        $this->render(
+            'index',
+            [
                 'model'         => $model,
                 'massiveAction' => $massiveAction,
                 'pageTitle'     => gT('User roles'),
-            )
-        );
-    }
-
-    /**
-     * Displays a particular model.
-     * @param integer $ptid the ID of the model to be displayed
-     */
-    public function viewrole($ptid)
-    {
-        if (!Permission::model()->hasGlobalPermission('superadmin', 'read')) {
-            Yii::app()->session['flashmessage'] = gT('You have no access to the role management!');
-            $this->getController()->redirect(array('/admin'));
-        }
-        $oPermissionTemplate = Permissiontemplates::model()->findByPk($ptid);
-        return $this->getController()->renderPartial(
-            '/admin/permissiontemplates/partials/_view',
-            [
-                "oModel" => $oPermissionTemplate,
             ]
         );
     }
 
+
     /**
-     * Edit Role Modal.
+     * Returns the modal view for adding/editing a user role
+     *
+     * @param null $ptid
```

Comment:
"null" is not a type per se. Maybe you meant "?int" or "int|null"?

Reply:
Done.